### DISCUSSION TITLE
241119_화

### DISCUSSION THREAD

**Discussion by Soy17** [boostcampaitech7/level2-cv-semanticsegmentation-cv-19-lv3#31] [TIL]
- [ ] UNet++ 관련 Augmentation 실험 진행
- [x] 진짜진짜강의막차

**Comment by june21a** [comment]
- [x] overlap 영역에 대한 EDA 수행하기
- [x] overlap 영역에 대한 segmentation을 수행함에 있어서 baseline 코드와 mmseg의 차이점 분석
- [x] 2번을 바탕으로 차이점이 있으면 그걸 반영해서 mmseg를 활용할 수 있도록 노력해보기..
- [x] 과제 마무리

**Comment by sejongmin** [comment]
- [ ] KFOLD 진행하기
- [x] SAM2-UNet 구축해보기

**Comment by Yoon0717** [comment]
- [x] Modularize 추가 진행
- [x] 다양한 Augmentation 진행

**Reply by minrongtic** [reply]
참 잘했어요~!

**Reply by Soy17** [reply]
참 잘했어요!

**Comment by wonjeongjeong** [comment]
- [x] 강의 9, 10 듣기
- [x] 과제 1~ 3 하기
- [ ] 강의에 나왔던 논문 하나 구현 시작

**Comment by minrongtic** [comment]
**TO DO** 

어제 자  #20 에 이어서 체크

- [ ] 시드 2024값 비교 
- [ ] 2048 돌아가는 백본 찾아보기
